fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -142,10 +147,12 @@ private CacheEntry(long length, List<ByteBuffer> buffers) {
     private static class DiskCacheEntry {
         private final long length;
         private final InputFile inputFile;
+        private int useCount;
```

Comment:
atomic int?

Reply:
I think there is no need for that. we use cache.asMap.computeIfPresent that is thread-safe for the cache. so pin() and unpin() are both safe. as discussed by https://github.com/ben-manes/caffeine/issues/513 compute will replace the entry with the compute() result, and Weights are measured and recorded when entries are inserted into or updated in the cache, and are thus effectively static during the lifetime of a cache entry, the weight is compute when insert, so use in weigher is also safe.